app/konnect/dev-portal/access-and-approval/manage-teams.md

```diff
@@ -0,0 +1,192 @@
+1. Create the `v1` version for the `Pizza Ordering` API product and publish it:
+```bash
+curl --request POST \
+  --url https://<region>.api.konghq.com/v2/api-products/{api-product-id}/product-versions \
+  --data "id": "GATEWAY_SERVICE_ID" \
+  --data "name": "v1" \
+  --data "gateway_service": {
+      "runtime_group_id": "RUNTIME_GROUP_ID"
+    } \
+  --data "publish_status": "published"
+```
+{:.note}
+> The `api-product-id` is the UUID of the Pizza Ordering API product.
+{% endnavtab %}
+{% endnavtabs %}
+
+### Create developer teams
+
+Next, let's create two developer teams: "Authorized Delivery Partners" with API Viewer and Consumer permissions and "Prospective Partners" with API Consumer permissions.
+
+{% navtabs %}
+{% navtab Konnect UI %}
+1. From the navigation menu, open {% konnect_icon dev-portal %} **Dev Portal**.
+1. Click **Teams** in the sidebar and then click **Add team**.
+1. Enter the following:
+    * **Team name:** Authorized Delivery Partners
+    * **Description:** Team of vetted delivery partners that can view and consume APIs
+1. On the **Developers** tab of your new team, click **Add developer** and select a developer to add to the team.
+1. On the **Products** tab of your new team, click **Add Role**.
+1. Select the **Pizza Ordering** product from the dropdown menu and select **API Consumer** and **API Viewer** from the **Add roles** dropdown menu.
+1. Click **Teams** in the sidebar and click **New Team** again and enter the following:
+    * **Team name:** Prospective Partners
+    * **Description:** Team of unvetted partners that can only view APIs
+1. On the **Developers** tab of your new team, click **Add developer** and select a developer to add to the team.
+1. On the **Products** tab of your new team, click **Add Role**.
+1. Select the **Pizza Ordering** product from the dropdown menu and select **API Viewer** from the **Add roles** dropdown menu. 
+{% endnavtab %}
+{% navtab API %}
+{:.note}
+> **Note:** Make sure to include headers with the personal access token in your requests.
+
+These instructions only use the required fields in the following API specs:
+* [Create developer team](https://developer.konghq.com/spec/2dad627f-7269-40db-ab14-01264379cec7/0ecb66fc-0049-414a-a1f9-f29e8a02c696#/Teams/create-portal-team)
+* [Assign roles to a team](https://developer.konghq.com/spec/2dad627f-7269-40db-ab14-01264379cec7/0ecb66fc-0049-414a-a1f9-f29e8a02c696#/Team%20Roles/portal-teams-assign-role)
+
+If you want to use additional fields, see the API spec documentation.
+
+1. Create the `Authorized Delivery Partners` team:
+```bash
+curl --request POST \
+  --url https://<region>.api.konghq.com/v2/portals/{portal-id}/teams \
+  --data "name": "Authorized Delivery Partners" \
+  --data "description": "Team of vetted delivery partners that can view and consume APIs"
+```
+  The `portal-id` will be the value from the output of the request you used earlier to create the Pizza Delivery API product.
```

Comment:
```suggestion
  The `portal-id` will be the value from the output of the request you used earlier to create the Pizza Delivery API product.
```

I can't find this portal-id output from any of the steps listed here. When you create the portal you get a portal ID, It's available in konnect manager and you can use it for future requests, but so far nothing has output for me, at least not automatically.